Submitted By:            Igor Živković <contact at igor hyphen zivkovic dot from dot hr>
Date:                    2013-07-28
Initial Package Version: 2.7
Upstream Status:         Fixed
Origin:                  Debian
Description:             Prevents installation of mp4ff header and library files
                         as its usage outside FAAD is discouraged.

diff -Naur faad2-2.7.orig/common/mp4ff/Makefile.am faad2-2.7/common/mp4ff/Makefile.am
--- faad2-2.7.orig/common/mp4ff/Makefile.am	2009-02-06 15:24:21.000000000 +0100
+++ faad2-2.7/common/mp4ff/Makefile.am	2013-07-28 15:08:46.363310961 +0200
@@ -1,5 +1,5 @@
-lib_LIBRARIES = libmp4ff.a
-include_HEADERS = mp4ff.h mp4ffint.h
+noinst_LIBRARIES = libmp4ff.a
+noinst_HEADERS = mp4ff.h mp4ffint.h
 
 libmp4ff_a_CFLAGS = -DUSE_TAGGING=1
 
